### ISSUE TITLE
Configure screenshots_dir in code, expose savedScreenshots

### ISSUE THREAD

**Issue by tacman** [zenstruck/browser#148]
I want to use this awesome library to navigate my site and save screenshots, not necessarily as part of a functional test, but just a regular app.

```php
final class AppScreenshotsCommand extends InvokableServiceCommand
{
    use PantherTestCaseTrait;
    use HasBrowser;

    public function __invoke(
        IO $io,
    ): void {
        $browser = $this->pantherBrowser(['screenshot_dir' => './screenshots']);
        $browser->visit('/')
            ->wait(4000) // wait for css??
            ->takeScreenshot('homepage.jpg');
        dd($browser->getSavedScreenshots());
```

How can I set the screenshot_dir in my console command?  The above does not work, $options is different in that call.

Also, to get the screenshots at the end, I needed to add a getter to the PantherBrowser.php, would you accept a PR for that?

```php
    public function getSavedScreenshots(): array
    {
        return $this->savedScreenshots;
    }
```
 
I am trying to use this to solve the problem of creating screenshots requiring authenticated users, e.g. 

https://github.com/Spomky-Labs/pwa-bundle/issues/134

**Comment by tacman** [comment]
There's  probably a more elegant way, but I fixed the directory like this:

```php
        $_SERVER['BROWSER_SCREENSHOT_DIR'] = './screenshots';
        $browser = $this->pantherBrowser();
```

**Comment by kbond** [comment]
> `$_SERVER['BROWSER_SCREENSHOT_DIR'] = './screenshots';`

Yeah, that's the way currently. The env var options got a bit out of control as I added features - I'd like to normalize this at some point.

> I needed to add a getter to the PantherBrowser.php, would you accept a PR for that?

Yep, sure!

**Comment by tacman** [comment]
Hmm, I'm doing something wrong, even waiting 8 seconds I'm not seeing any of the css applied.

FWIW, I'm in dev and using assetmapper.

```
        $_SERVER['BROWSER_SCREENSHOT_DIR'] = './screenshots';
        $_SERVER['PANTHER_CHROME_ARGUMENTS'] = "--disable-dev-shm-usage --window-size=600,800 --no-sandbox --headless --hide-scrollbars";

        $browser = $this->pantherBrowser();
        $browser->visit('/')
            ->wait(8000)
            ->takeScreenshot('homepage.jpg');
```


![homepage](https://github.com/user-attachments/assets/dac64fa1-b85f-469a-a964-ab00034f9bc4)

symfony open:local works quickly and as expected:

![image](https://github.com/user-attachments/assets/db72abb0-cb3c-4b59-89cc-1f8bc5d3ca2b)
